### PR TITLE
libdef and tests for read-last-lines npm module

### PR DESCRIPTION
**Pull request by charlieblevins**
Please consider this type definition for read-last-lines. Apologies in advance for missing any details - this is my first pull request.

### CODE REVIEW

File: definitions/npm/read-last-lines_v1.x.x/flow_v0.25.x-/read-last-lines_v1.x.x.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// @flow
+declare module 'read-last-lines' {
+    declare module.exports: {
+        read(input_file_path: string, maxLineCount: number, encoding: ?string): Promise<string>
+    };
+}
```
Comment (GAntoine) [inline definitions/npm/read-last-lines_v1.x.x/flow_v0.25.x-/read-last-lines_v1.x.x.js]: Add a newline at EOF

File: definitions/npm/read-last-lines_v1.x.x/flow_v0.25.x-/read-last-lines_v1.x.x.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// @flow
+declare module 'read-last-lines' {
+    declare module.exports: {
+        read(input_file_path: string, maxLineCount: number, encoding: ?string): Promise<string>
```
Comment (GAntoine) [inline definitions/npm/read-last-lines_v1.x.x/flow_v0.25.x-/read-last-lines_v1.x.x.js]: If `encoding` is optional, you want `encoding?: string`

File: definitions/npm/read-last-lines_v1.x.x/flow_v0.25.x-/test_read-last-lines_v1.x.x.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+// @flow
+console.log('TEST');
+import { describe, it } from 'flow-typed-test';
+const rll = require('read-last-lines');
+describe('read', (done) => {
+
```
Comment (GAntoine) [inline definitions/npm/read-last-lines_v1.x.x/flow_v0.25.x-/test_read-last-lines_v1.x.x.js]: Remove extra empty line

File: definitions/npm/read-last-lines_v1.x.x/flow_v0.25.x-/test_read-last-lines_v1.x.x.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+// @flow
+console.log('TEST');
```
Comment (GAntoine) [inline definitions/npm/read-last-lines_v1.x.x/flow_v0.25.x-/test_read-last-lines_v1.x.x.js]: Remove console.log

File: definitions/npm/read-last-lines_v1.x.x/flow_v0.25.x-/test_read-last-lines_v1.x.x.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+import { describe, it } from 'flow-typed-test';
+const rll = require('read-last-lines');
```
Comment (GAntoine) [inline definitions/npm/read-last-lines_v1.x.x/flow_v0.25.x-/test_read-last-lines_v1.x.x.js]: Use an import statement `import rll from 'read-last-lines'`

File: definitions/npm/read-last-lines_v1.x.x/flow_v0.25.x-/test_read-last-lines_v1.x.x.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+    rll.read('read-last-lines_v1.x.x.js', 25);
+    rll.read('read-last-lines_v1.x.x.js', 25, 'utf-8');
+    rll.read('read-last-lines_v1.x.x.js', 25, 'buffer');
+
+    // $ExpectError
+    rll.read();
+    // $ExpectError
+    rll.read('some-file');
+});
```
Comment (GAntoine) [inline definitions/npm/read-last-lines_v1.x.x/flow_v0.25.x-/test_read-last-lines_v1.x.x.js]: Add newline

**Comment by charlieblevins** [comment]
Thanks @pascalduez, changes made

**Pull request merged**